starknet-foundry/crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -0,0 +1,187 @@
+    let private_key = SigningKey::from_random();
+    let public_key = private_key.verifying_key();
+    let salt = match maybe_salt {
+        Some(salt) => salt,
+        None => FieldElement::from(OsRng.next_u64()),
+    };
+
+    let address = get_contract_address(
+        salt,
+        parse_number(OZ_CLASS_HASH)?,
+        &[public_key.scalar()],
+        FieldElement::ZERO,
+    );
+
+    let max_fee = {
+        let signer = LocalWallet::from_signing_key(private_key.clone());
+        let factory = OpenZeppelinAccountFactory::new(
+            parse_number(OZ_CLASS_HASH)?,
+            get_network(network)?.get_chain_id(),
+            signer,
+            provider,
+        )
+        .await?;
+        let deployment = factory.deploy(salt);
+
+        deployment.estimate_fee().await?.overall_fee
+    };
+
+    if !accounts_file_path.exists() {
+        std::fs::create_dir_all(accounts_file_path.clone().parent().unwrap())?;
+        std::fs::write(accounts_file_path.clone(), "{}")?;
+    }
+
+    let contents = std::fs::read_to_string(accounts_file_path.clone())?;
+    let mut items: serde_json::Value =
+        serde_json::from_str(&contents).expect("failed to parse json file");
+
+    let network_value = get_network(network)?.get_value();
+
+    if !items[network_value][&name].is_null() {
+        return Err(anyhow!(
+            "Account with provided name already exists in this network"
+        ));
+    }
+
+    items[network_value][&name] = json!({
+        "private_key": format!("{:#x}", private_key.secret_scalar()),
+        "public_key": format!("{:#x}", public_key.scalar()),
+        "address": format!("{address:#x}"),
+        "salt": format!("{salt:#x}"),
+        "deployed": false,
+    });
+
+    std::fs::write(
+        accounts_file_path.clone(),
+        serde_json::to_string_pretty(&items).unwrap(),
+    )?;
+
+    let mut output = vec![(
+        "message",
+        format!("Account successfully created. Prefund generated address with at least {max_fee} tokens. \
+         It is good to send more in the case of higher demand, max_fee * 2 = {}", max_fee * 2),
+    ), ("address", format!("{address:#x}"))];
+    if add_profile {
+        output.push((
+            "add_profile",
+            add_created_profile_to_configuration(name, network.to_string(), url),
+        ));
+    }
+
+    Ok(output)
+}
+
+pub fn print_account_create_result(
+    account_create_result: Result<Vec<(&'static str, String)>>,
+    int_format: bool,
+    json: bool,
+) -> Result<()> {
+    match account_create_result {
+        Ok(mut values) => {
+            values.insert(0, ("command", "Create account".to_string()));
+            print_formatted(values, int_format, json, false)?;
+        }
+        Err(error) => {
+            print_formatted(vec![("error", error.to_string())], int_format, json, true)?;
+        }
+    };
+
+    Ok(())
+}
+
+pub fn add_created_profile_to_configuration(name: String, network: String, url: String) -> String {
```

Comment:
```suggestion
pub fn add_created_profile_to_configuration(name: String, network: String, url: String) -> Result<String> {
```
why do you return string, not result?